client_it_test.go

```diff
@@ -18,6 +18,8 @@ package hazelcast_test
 
 import (
 	"context"
+	"fmt"
+	"github.com/hazelcast/hazelcast-go-client/internal"
 	"reflect"
 	"sync"
 	"sync/atomic"
```

Comment:
Can you organize these imports so imports from standard lib are separated from others with a blank line?